securedrop/debian/rules

```diff
@@ -63,11 +63,11 @@ override_dh_installinit:
 	dh_installinit --noscripts
 
 override_dh_systemd_enable:
-	dh_systemd_enable --name=securedrop_rqrequeue
-	dh_systemd_enable --name=securedrop_rqworker
-	dh_systemd_enable --name=securedrop_shredder
+	dh_systemd_enable --no-enable securedrop-submissions-today.service
+	dh_systemd_enable --no-enable securedrop-clean-tmp.service
+	dh_systemd_enable
 
 override_dh_systemd_start:
-	dh_systemd_start --name=securedrop_rqrequeue
-	dh_systemd_start --name=securedrop_rqworker
-	dh_systemd_start --name=securedrop_shredder
+	dh_systemd_start --no-start securedrop-submissions-today.service
+	dh_systemd_start --no-start securedrop-clean-tmp.service
+	dh_systemd_start
```

Comment:
`dh_systemd_{enable,start} --no-{enable,start}` is a bit of a riddle.  Would you mind documenting these overrides?  :-)

Reply:
Yes...